compiler/tests/testSuite.bal

```diff
@@ -212,8 +212,8 @@ function resolveTestSemtype(t:Context tc, map<t:SemType> m, s:Identifier|s:TypeP
         int|s:Identifier index = tn.index;
         if t:isSubtypeSimple(t, t:LIST) {
             if index is int {
-                return t:listMemberType(tc, t, index);
-                //return t:listProj(tc, t, index);
+                //return t:listMemberType(tc, t, index);
+                return t:listProj(tc, t, index);
```

Comment:
We need to test listMemberType too. I suggest:

- call both (in cases where index is a singleton)
- check that result from listProj is right
- check that result from listMemberType is a supertype of result from listProj